app/controllers/buyers/invoices_controller.rb

```diff
@@ -46,6 +47,10 @@ def update
   end
 
   private
+  def authorize_finance
+    authorize! :read, :finance
```

Comment:
It shouldn't be always 'read'. One can have permission to read but not to update for example.

Reply:
@Martouta For show it makes sense no?